src/plugins/home/README.md

```diff
@@ -53,6 +53,6 @@ Follow the steps below to add new Sample data sets to OpenSearch Dashboards.
 
 1. Create new-line delimited json containing sample data.
 2. Create file with OpenSearch field mappings for sample data indices.
-3. Create OpenSearch Dashboards saved objects for sample data including index-patterns, visualizations, and dashboards. The best way to extract the saved objects is from the OpenSearch Dashboards management -> saved objects [export UI](https://www.opensearch.org/guide/en/kibana/current/managing-saved-objects.html#_export)
+3. Create OpenSearch Dashboards saved objects for sample data including index-patterns, visualizations, and dashboards. The best way to extract the saved objects is from the OpenSearch Dashboards management -> saved objects [export UI](https://www.elastic.co/guide/en/kibana/current/managing-saved-objects.html#_export)
```

Comment:
Remove this or link it somewhere else.

Reply:
I think in this case we should remove the `export UI` link and add it to the `noDocumentation` list.